src/views/Playgrounds/PlaygroundForm.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react'
+import { useForm, Controller } from 'react-hook-form'
+import {
+  Button as _Button,
+  Text,
+  FormField,
+  TextField,
+  Select,
+  Checkbox,
+  Textarea,
+  HeaderTextField,
+} from '@/shared/components'
+import { textFieldValidation, requiredValidation } from './formValidationOptions'
+import { SelectedItem } from '@/shared/components/Select'
+import styled from '@emotion/styled'
+
+const items: SelectedItem[] = [
+  { name: 'Public (Anyone can see this video', value: 'public' },
+  { name: 'Private', value: 'private' },
+]
+
+type Inputs = {
+  title: string
+  videoSelect: string
```

Comment:
```suggestion
  selectedVideo: string
```
or something similar